### DISCUSSION TITLE
bug: cannot specify the required methods for WalletConnect

### DISCUSSION THREAD

**Discussion by maxaleks** [wagmi-dev/references#378] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently the default list of required methods is used

### Expected Behavior

Allow to pass the list of required methods to the WalletConnectConnector

**Comment by tmm** [comment]
Moved to feature request.

What methods were you hoping to add? This can effect the behavior of how the connector works within wagmi so want to learn more about what you are trying to do.

**Reply by maxaleks** [reply]
This issue is a bit outdated 🫠 Things have been changing so fast in the last couple of days around WalletConnect. Initially the problem was that when I connected the wallet through WalletConnect v2, some of the methods I needed were missing. For example, AlphaWallet only supported `eth_sendTransaction` and `personal_sign`, but I needed `eth_signTypedData_v4`. Now all methods are supported. I don't know on whose side the changes were made, but for now we can close this issue.

**Reply by tmm** [reply]
Sounds good! Appreciate the patience.

**Comment by percyzt** [comment]
wallet connnect v2    --> Scanning the code will flash back, what is the reason for this?
